src/yunohost/app.py

```diff
@@ -2439,9 +2439,16 @@ def _parse_args_in_yunohost_format(args, action_args):
 
                 elif arg_type == 'user':
                     msignals.display(m18n.n('users_available'))
-                    for user in user_list()['users'].keys():
+                    users = user_list()['users']
+                    for user in users.keys():
                         msignals.display("- {}".format(user))
 
+                    root_mail = "root@%s" % _get_maindomain()
+                    for user in users.keys():
+                        if root_mail in user_info(user)["mail-aliases"]:
```

Comment:
is it possible to have several users with this aliase?

Reply:
I don't think so ?